test/suite/Annotations.test.php

```diff
@@ -580,17 +587,75 @@ protected function testGetAnnotationsFromAnInterface()
 
     protected function testGetAnnotationsFromTrait()
     {
-        if (version_compare(PHP_VERSION, '5.4.0', '<')) {
```

Comment:
Isn't this completely redundant? The version constraint in Composer will take care of this.